test/Azure.Functions.Cli.Tests/Resources/DurableCreateHelperTestProjects/AttributeMatchingTest/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do you need a `.gitignore` per scenario here? seems a bit too much :)

Reply:
Agreed! I'll look at moving that up a level to avoid one per scenario

Reply:
This is done in the latest commits